pkg/network/networker.go

```diff
@@ -415,17 +415,13 @@ nft 'add rule bridge filter prerouting iifname "{{.Iface}}" jump {{.Name}}-pre'
 nft 'add rule bridge filter postrouting oifname "{{.Iface}}" jump {{.Name}}-post'
 
 nft 'add rule bridge filter {{.Name}}-pre ip saddr . ether saddr != { {{.IPv4}} . {{.Mac}} } counter drop'
-{{if .IPv6}}
 nft 'add rule bridge filter {{.Name}}-pre ip6 saddr . ether saddr != { {{.IPv6}} . {{.Mac}} } counter drop'
-{{end}}
 
 nft 'add rule bridge filter {{.Name}}-pre arp operation reply arp saddr ip != {{.IPv4}} counter drop'
 nft 'add rule bridge filter {{.Name}}-pre arp operation request arp saddr ip != {{.IPv4}} counter drop'
```

Comment:
Should similar rules for ipv6 be added to prevent taking over an unassigned ip? Also, the scenario in this [issue](https://github.com/threefoldtech/zos/issues/1457) works with ygg ips.

Reply:
No, as far as I remember this filter for arp was needed for ipv4.